### PR TITLE
feat(threaded-replies): keep replies visible when loading more replies

### PR DESCRIPTION
**Pull request by wpiesiak**

### CODE REVIEW

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityThreadReplies.js [hunk 1]
```diff
@@ -43,6 +46,11 @@ const ActivityThreadReplies = ({
 
     return (
         <div className="bcs-ActivityThreadReplies" data-testid="activity-thread-replies">
+            {isRepliesLoading && (
+                <div className="bcs-ActivityThread-loading" data-testid="activity-thread-loading">
```
Comment (jstoffan) [inline src/elements/content-sidebar/activity-feed/activity-feed/ActivityThreadReplies.js:50]: Can we update the class names and testid attributes?

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityThreadReplies.js [hunk 2]
```diff
@@ -2,6 +2,7 @@
 import React from 'react';
 import Comment from '../comment';
 
+import LoadingIndicator from '../../../../components/loading-indicator';
```
Comment (jstoffan) [inline src/elements/content-sidebar/activity-feed/activity-feed/ActivityThreadReplies.js:5]: Should this be grouped with the `Comment` import above?

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.js [hunk 3]
```diff
@@ -113,19 +112,15 @@ const ActivityThread = ({
         <div className="bcs-ActivityThread" data-testid="activity-thread">
             {children}
 
-            {isRepliesLoading && (
-                <div className="bcs-ActivityThread-loading" data-testid="activity-thread-loading">
```
Comment (jstoffan) [inline src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.js:117]: Does the styling for this also need to move to the `ActivityThreadReplies` stylesheet?

File: src/elements/content-sidebar/activity-feed/activity-feed/__tests__/ActivityThreadReplies.test.js [hunk 4]
```diff
@@ -14,4 +14,10 @@ describe('src/elements/content-sidebar/activity-feed/activity-feed/ActivityThrea
         expect(queryByText(replies[1].message)).toBeVisible();
         expect(queryByText(replies[0].message)).toBeVisible();
     });
+
+    test('should render loading indicator if isRepliesLoading is true', () => {
+        const { queryByTestId } = getWrapper({ isRepliesLoading: true });
```
Comment (jstoffan) [inline src/elements/content-sidebar/activity-feed/activity-feed/__tests__/ActivityThreadReplies.test.js:19]: Do we have a negative test case?

Comment (wpiesiak) [reply src/elements/content-sidebar/activity-feed/activity-feed/__tests__/ActivityThreadReplies.test.js:19]: added 👍

**Pull request merged**